bedevere/files.py

```diff
@@ -0,0 +1,20 @@
+"""Checks related to files on a pull request."""
+import gidgethub.routing
+
+from .news import check_news
+from .prtype import classify_by_files
+from . import util
+
+
+router = gidgethub.routing.Router()
+
+
+@router.register('pull_request', action='opened')
+@router.register('pull_request', action='synchronize')
+async def retrieve_files(event, gh, *args, **kwargs):
```

Comment:
This is doing more than just retrieving the files list, so the function name is misleading. Maybe `check_file_paths()`?

Reply:
Thanks!